python/src/aiconfig/editor/server/server.py

```diff
@@ -109,27 +110,88 @@ def _get_server_state(app: Flask) -> ServerState:
 
 @app.route("/")
 def home():
-    ss = _get_server_state(app)
-    ss.count += 1
-    LOGGER.info("Count: %s", ss.count)
     return app.send_static_file("index.html")
 
 
-@app.route("/test")
-def test():
-    return {"key": 6}
+@app.route("/api/load", methods=["POST"])
+def load():
+    ss = _get_server_state(app)
+    request_json = request.get_json()
+    path = request_json["path"]
+    LOGGER.info(f"Loading AIConfig from {path}")
+    if not path:
+        return {"message": "No path provided"}, 400
+    elif not os.path.isfile(path):
+        return {"message": f"File does not exist: {path}"}, 400
+    else:
+        try:
+            ss.aiconfig_runtime = AIConfigRuntime.load(path)  # type: ignore
+            return {"message": "Done"}, 200
+        except Exception as e:
+            return {"message": f"<p>Failed to load AIConfig from {path}: {e}"}, 400
+
+
+@app.route("/api/save", methods=["POST"])
+def save():
+    ss = _get_server_state(app)
+    request_json = request.get_json()
+    path = request_json["path"]
+    LOGGER.info(f"Saving AIConfig to {path}")
+    if not path:
+        return {"message": "No path provided"}, 400
+    else:
+        try:
+            ss.aiconfig_runtime.save(path)  # type: ignore
+            return {"message": "Done"}, 200
+        except Exception as e:
+            err: Err[str] = core_utils.ErrWithTraceback(e)
+            return {"message": f"<p>Failed to save AIConfig to {path}: {err}"}, 400
+
+
+@app.route("/api/create", methods=["POST"])
+def create():
+    ss = _get_server_state(app)
+    ss.aiconfig_runtime = AIConfigRuntime.create()  # type: ignore
+    return {"message": "Done"}, 200
 
 
-@app.route("/api/run")
+@app.route("/api/run", methods=["POST"])
 async def run():
     ss = _get_server_state(app)
-    result = await ss.aiconfig_runtime.run(
-        "gen_packing_list",
-        #
-        params={"location": "central park"},
-    )
-    text = result.data[0]
-    return f"<p>{text}</p>"
+    request_json = request.get_json()
+    prompt_name = request_json.get("prompt_name", None)
+    stream = request_json.get("stream", True)
+    LOGGER.info(f"Running prompt: {prompt_name}, {stream=}")
+    inference_options = InferenceOptions(stream=stream)
+    try:
+        result = await ss.aiconfig_runtime.run(prompt_name, options=inference_options)  # type: ignore
+        LOGGER.debug(f"Result: {result=}")
+        result_text = str(
+            ss.aiconfig_runtime.get_output_text(prompt_name)  # type: ignore
+            #
```

Comment:
Remove?

Reply:
I'm using these # lines as a hack because otherwise it autoformats to very long lines...